src/util/sandbox/fd_sandbox_private.h

```diff
@@ -0,0 +1,17 @@
+#ifdef FD_HAS_SANDBOX
+
+#ifndef HEADER_fd_src_util_sandbox_fd_sandbox_private_h
+#define HEADER_fd_src_util_sandbox_fd_sandbox_private_h
+
+#include <linux/filter.h>
```

Comment:
A private internal header makes more sense here for an include like this.  But this is also redundant with the previous header.  And generally private headers are used for all sorts of implementation specific details that should not be public but are needed to coordinate across multiple compile units.  So, there is a good chance this doesn't need to be here either as per the previous discussion.